test/test_encryption.py

```diff
@@ -2099,5 +2099,92 @@ def test_05_roundtrip_encrypted_unindexed(self):
         self.assertEqual(decrypted, val)
 
 
+class TestQueryableEncryptionDocsExample(EncryptionIntegrationTest):
+    # Queryable Encryption is not supported on Standalone topology.
+    @client_context.require_no_standalone
+    @client_context.require_version_min(6, 0, -1)
+    def setUp(self):
+        super().setUp()
+
+    def test_queryable_encryption(self):
+        # Drop data from prior test runs.
+        MongoClient().keyvault.datakeys.drop()
+        MongoClient().docs_examples.encrypted.drop()
```

Comment:
These should use the existing self.client. Also do we need to pass encrypted_fields to drop the metadata collections?

Reply:
Done, and updated to remove the whole database.